src/UserNSRunner.jl

```diff
@@ -62,6 +62,21 @@ function UserNSRunner(workspace_root::String;
         push!(workspaces, ccache_dir() => "/root/.ccache")
     end
 
+    # If we are on a system that uses `/etc/resolv.conf` as a resolver
+    # configuration file (mainly *BSD and Linux), check if a nameserver is
+    # specified and if yes, mount the existing `/etc/resolv.conf` to make use
+    # of system-specific nameserver settings
+    if (Sys.isbsd() || Sys.islinux()) && isfile("/etc/resolv.conf")
+      if any(startswith(line, "nameserver") for line in eachline("/etc/resolv.conf"))
+        # We copy the contents of `/etc/resolv.conf` to a temporary file and mount that one,
+        # such that `/etc/resolv.conf` is editable inside the container
+        tmppath, tmpio = mktemp()
+        write(tmpio, read("/etc/resolv.conf", String))
+        flush(tmpio) # required as otherwise `tmppath` remains empty
+        push!(workspaces, tmppath => "/etc/resolv.conf")
+      end
+    end
```

Comment:
this is really nitpicky, but can you indent 4 spaces instead of just 2 to match the rest of the file?  Thanks!

Reply:
Sorry, my bad. Fixed in f065e57.